### DISCUSSION TITLE
NASA's Operation IceBridge Aircraft Missions

### DISCUSSION THREAD

**Discussion by siddhi1991** [qgreenland-workshop-2023-researcher/qgreenland-workshop-2023-researcher.github.io#133] [Dataset Ideas]
### Dataset URL

https://nsidc.org/data/icebridge

### Description

Operation IceBridge is an initiative by NASA, in which several airborne surveys were conducted to map coastal Greenland, coastal Antarctica, the Antarctic Peninsula, interior Antarctica, the southeast Alaskan glaciers, and Antarctic and Arctic sea ice during 2009-2020.  The operation IceBridge mission utilized >1000 aircrafts fitted with a suite of instruments including radar sounders, gravimeters, magnetometers and mapping cameras for mapping the polar ice sheets and glaciers. Besides spatial maps, the dataset also provides information on the topography of bedrock under ice sheets, glacier retreat, the sea ice elevation changes, glacier and ice shelf grounding lines, sea ice distribution, sea ice freeboard, ice temperature, and meteorological observations.

**Comment by lauralindzey** [comment]
Hopefully you and the poster of #142 are in the same breakout group ;-)

I'm also interested in the OIB datasets, specifically the airborne component.
I've put a geopackage file in group four's directory containing all of the CReSIS flightlines in Greenland, which covers most (all?) of the Greenland OIB airborne data, as well as some other airborne data in the arctic. (UTIG did some OIB flights in Antarctica; I'm not sure if they had any in Greenland.)
